README.md

```diff
@@ -38,16 +38,21 @@ Enjoy your new RPM.
 
 ### Ubuntu/Debian
 
-Building a .deb should be straight forward with:
+On recent Debian/Ubuntu distributions you should be able to install with:
+```
+# apt-get install TODO
```

Comment:
intltool,
libgnome-keyring-dev,
libgtk-3-dev (>= 3.4),
libnm-glib-vpn-dev (>= 1.1.0),
libnm-gtk-dev (>= 1.1.0),
libnm-util-dev (>= 1.1.0),
libnma-dev (>= 1.1.0),
libsecret-1-dev (>= 0.18),

Should build in current testing or newer.
1.2.0 tag is already in testing btw.

May 10 2016 2:37 PM, "Dan Fruehauf"  wrote:
	In README.md (https://github.com/danfruehauf/NetworkManager-ssh/pull/56#discussion_r62663053):

	> @@ -38,16 +38,21 @@ Enjoy your new RPM. > > ### Ubuntu/Debian > > -Building a .deb should be straight forward with: > +On recent Debian/Ubuntu distributions you should be able to install with: > +``` > +# apt-get install TODO

	Can you please provide me with the name of the packages that users should install on Debian/Ubuntu? Thanks!

	—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub (https://github.com/danfruehauf/NetworkManager-ssh/pull/56/files/0c5014df9a828407a4f011add5b78720f917856c#r62663053)